### ISSUE TITLE
Request  Merging of most of the best features of currently updated floris mod into dickplomacy reloaded

### ISSUE THREAD

**Issue by WHYDONTYOULIKETHEPIECEAPIE** [LilyModzStuff/warband_mod_source#56]
I don't know how difficult this would be, but I'm only assuming from the perspective of someone who doesn't know much about coding that it may be easier to get permission and moves some good stuff over and patch it in than it would be to make all of the from nothing, so maybe this isn't too difficult?

I Just really really felt like this is the right thing to do at this point.

I'm just completely torn between playing the floris mod and playing dickplomacy reloaded, it hurts really really bad! I feel like the floris mod pack and dickplomacy reloaded are like two different huge good mods for mount and blade warband, but oh gosh if only dickplomacy could stay "dickplomacy" but just get all the extra good stuff from floris? There's people who don't care about the sexy stuff from dickplomacy, so they could just keep playing floris, but for the people who love the sexy stuff, maybe we could have a "Dickflorplomacy reloaded" or a "Flordickplomacy reloaed" or a "Dickplomacy re-florded" ?

It would be SOOOOO coool! it would just take a few movings arounds of files and asking the different mod people if it's okay with them and then this mod could be like WHOOAAAAA.

PLEASE please PLEASE help make this happen! I don't want to change floris, I just want dickplomacy reloaded to have all the good extra stuff from floris added in! (I'm talking about floris "Expanded" of course because that has tons of good stuff) maybe with optional settings for those who need less intense computer stuff. that way everyone wins! I beg you. Please please please please please do this!

https://www.moddb.com/mods/floris-mod-pack
> "Permissions for using Floris as a basis for your mod
> The Floris Mod Pack is a compilation of a lot of other mods, along with a lot of our own work. We not only provide the source, but give you the possibility with the Dev Suite to recompile the entire mod with your own adjustments. Of course we'd like to see the fruitations of your work, and incorporate good ideas!
> 
> But when you publish a mod that uses resources from other people, you need to have permission. A lot of the resources that are used in Floris are OSP (Open Source Project): this means that you can use these resources freely for non-commercial use, as long as you give the original authors credit (altough I recommend checking their pages out: sometimes there are additional conditions, like informing the author that you use their mod). But not every mod used by Floris is OSP. This means that for some mods you need to ask permission from the original authors.
> 
> Of course, when you create just some new txt-files (like when you create a new troop tree, or adjust some item/troop stats, or make a new script, or make a translation) and just publish it like that, without the rest of the pack incorporated into your download, you don't have to ask permission from everyone. But if you do incorporate resources from the pack, you need to ask permission.This means that when you use resources from Floris, you can't just use the Dev Suite, make your own mod and publish it without going through the following steps:
> 
>     Check what you use. What items do you use? All of the ones from Floris? Or just a few? And what about the music, sounds, textures...
>     Check where it's from. From what mods did those items, textures, music, scripts and so on come?
>     Ask permission. If stuff you're using is coming from non-OSP mods, you'll need to obtain permission to use it. Sometimes this can take a while, for not every mod author checks the forum regularly.
>     Credit the mods and their authors. Once you've got permission, you should give credit where credit is due.
> 
> It can be daunting to check what item comes form where, but if you've got the names of the items/sounds/music tracks/etc you can always check with me (but do take into account that it might take some time for me to check). If you use everything from Floris, you can of course just ask permission from all the mod authors listed as non-OSP.
> The list with all the resources used in Floris, and thus what mods are OSP or not, can be found over here: Forums.taleworlds.com"

**Comment by Vexince** [comment]
ok so I have recently been trying to find people who also have similar ideas, but I just cant find anyone. I'm still not sure how this could be done but by writing this reply now maybe it could revive this forum again? I would really like to see this happen as I absolutely love both Floris Expanded and Diplomacy and would love to see them combined, especially Floris + CTT mod.

**Comment by FRA2121** [comment]
> 

Yes, i've been looking for it now and think this is basicaly a must...  i'm not a coder either but also hoping this revives the thread and maybe someone with skill and interest can do this?
